drivers/mm/Kconfig

```diff
@@ -26,6 +26,19 @@ config MM_DRV_INTEL_ADSP_MTL_TLB
 	  Driver for the translation lookup buffer on
 	  Intel Audio DSP hardware (Meteor Lake).
 
+if MM_DRV_INTEL_ADSP_MTL_TLB
+
+config MM_DRV_INTEL_ADSP_MTL_TLB_POWER_OFF_UNUSED_RAM
```

Comment:
this is a very long config name and should not have the SOC name to start with, you might depend on the SOC instead...